src/lib/gql/mutations/user.ts

```diff
@@ -0,0 +1,64 @@
+import { gql } from '@apollo/client'
+
+export const SIGN_IN = gql`
+  mutation signIn(
+    $email: String
+    $fullName: String
+    $avatar: String
+    $providerId: String
+  ) {
+    signIn(
+      email: $email
+      fullName: $fullName
+      avatar: $avatar
+      providerId: $providerId
+    ) {
+      ... on User {
+        _id
+        email
+        firstName
+        lastName
+        providerId
+        avatar
+        createdAt
+        lastLogin
+        newUser
+        tier
```

Comment:
Is this the wrong tier then ?

Reply:
Nope you right. This is the only spot thats correct